sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,29 @@
+--
+UPDATE `creature_template` SET `AIName` = 'SmartAI', `ScriptName` = '' WHERE `entry` IN (25967,23483,23484);
+
+INSERT INTO `gossip_menu` (`MenuID`, `TextID`, `VerifiedBuild`) VALUES
```

Comment:
Delete statement here.

Reply:
Ah, right. Add the missing `DELETE FROM` line above the `INSERT INTO` line (row).